ramls/circulation.raml

```diff
@@ -561,6 +561,22 @@ resourceTypes:
         responses:
           501:
             description: "Not implemented yet"
+      /move:
```

Comment:
I'm aware we have discussed this before, and I may not have understood properly.

My understanding is this process needs to know the `request being moved`, `destination item` and the (potentially changed) `request type`. The first of those being provided by the `requestId` in the path.

What other information is needed?

Reply:
I will gladly make any changes required. Requirements are great. Settling them in advance even better. But we do what we can.

Reply:
Now `/circulation/requests/{id}/move` `POST` request body is simplified as request-move with `destinationItemId` and optional `requestType`.